source/manual/fall-back-to-mirror.html.md

```diff
@@ -1,90 +1,83 @@
 ---
 owner_slack: "#govuk-2ndline"
-title: Fall back to the static mirrors
+title: GOV.UK CDN static mirrors
 section: Deployment
 layout: manual_layout
 parent: "/manual.html"
 ---
 
-We maintain a static copy of most of the site, which gets used by the content delivery
-network (CDN) whenever origin (the application server) times out or serves an error
-response.
+GOV.UK uses the [Fastly content delivery network (CDN)](/manual/cdn.html) to handle most requests to [GOV.UK Origin](/manual/architecture-shallow-dive.html#a-user-visits-the-gov-uk-homepage). However, user requests will not get to Fastly in the following scenarios:
 
-This process is handled by our CDN config and is entirely transparent to us and
-our users. It happens multiple times a day, for lots of different reasons.
+- first and most commonly, user requests to GOV.UK may time out or return an error response
+- second, if Fastly goes down, user requests to Fastly will fail with a 5xx error code (the problem was caused by the server)
 
-This is why we refer to switching off Nginx on the origin cache machines as
-"falling back to the mirrors".
+When user requests cannot get to Fastly, those requests will instead go to a static copy of GOV.UK known as the GOV.UK mirror.
```

Comment:
This isn't quite right. If Fastly goes down, no requests are ever going to reach any part of GOV.UK (i.e. neither the origin nor the mirror). We have to [manually fail over to AWS CloudFront](https://docs.publishing.service.gov.uk/manual/fall-back-to-aws-cloudfront.html) in such a scenario, so that user requests go to AWS rather than to Fastly. AWS will, in turn, make requests to the mirror.

Reply:
Thanks @ChrisBAshton. How about the following:

"When user requests to GOV.UK time out or return an error response, these requests instead go to a static copy of GOV.UK known as the GOV.UK mirror.

If Fastly has gone down, we [manually fall back to AWS CloudFront](/manual/fall-back-to-aws-cloudfront.html), the backup CDN. AWS will then send user requests to the GOV.UK mirror."